tests/functional/cylc-trigger/07-kill-trigger/reference.log

```diff
@@ -0,0 +1,2 @@
+1/a -triggered off [] in flow 1
+1/a -triggered off [] in flow 1
```

Comment:
The test is not doing a reftest though?

Reply:
Dammit, my fault, I forgot to turn this on in the functional test.

Reply:
FYI I had to tweak your func test slightly to make it work as a reference test:
```diff
 [scheduler]
     [[events]]
         inactivity timeout = PT1M
+        expected task failures = "1/a"

 [scheduling]
     [[graph]]
@@ -14,6 +15,8 @@
                 # then kill the job
                 cylc__job__poll_grep_workflow_log -E '1/a.*running'
                 cylc kill "${CYLC_WORKFLOW_ID}//1/a"
+                # do not succeed immediately after issuing the kill command or the
+                # workflow may shut down as complete before registering task failure
+                # (This polling grep will never complete, but you know what I mean.)
+                cylc__job__poll_grep_workflow_log -E '1/a.*failed'
             fi
         """
```